java/ct.proto

```diff
@@ -51,16 +51,6 @@ message PreCert {
   optional bytes tbs_certificate = 2;
```

Comment:
I'm surprised you're not ditching this message at the same time, as it's so tightly coupled.